src/libraries/System.DirectoryServices/ref/System.DirectoryServices.cs

```diff
@@ -137,17 +137,17 @@ public partial class DirectoryEntries : System.Collections.IEnumerable
         public System.DirectoryServices.SchemaNameCollection SchemaFilter { get { throw null; } }
         public System.DirectoryServices.DirectoryEntry Add(string name, string schemaClassName) { throw null; }
         public System.DirectoryServices.DirectoryEntry Find(string name) { throw null; }
-        public System.DirectoryServices.DirectoryEntry Find(string name, string schemaClassName) { throw null; }
+        public System.DirectoryServices.DirectoryEntry Find(string name, string? schemaClassName) { throw null; }
```

Comment:
schemaClassName can be null for Find but not for Add?

Reply:
Yes, I double checked that. For Find it means "I do not care about schemaClassName"